src/Test.FunctionalTests.Sarif/MultitoolCommandLineTests.cs

```diff
@@ -4,10 +4,9 @@
 using System.Diagnostics;
 using System.IO;
 using FluentAssertions;
-using Microsoft.CodeAnalysis.Sarif;
 using Xunit;
 
-namespace Microsoft.CodeAnalysis.Test.FunctionalTests.Sarif
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
```

Comment:
>Multitool [](start = 39, length = 9)

While I'm in here, clean up a few namespaces to conform to code base standard that test classes live in the same namespace as corresponding product code.

I did not make the same change in Test.UnitTests.Sarif, even though I added a test file there as well, because there would have been too many changes for this PR.